sequencing/ATACSeq/jobSubmission/1_batchRunPreAnalysis.sh

```diff
@@ -52,10 +48,9 @@ echo "Script is running from directory: " ${SCRIPTDIR}
 
 ## Log files directory
 LOG_DIR=ATACSeq/logFiles/${USER}/${SLURM_ARRAY_JOB_ID}
-
 echo "Log files will be moved to dir: " $LOG_DIR
 mkdir -p $LOG_DIR
-mv "./ATACSeq/logFiles/${USER}/ATACpreAnalysisS1${SLURM_ARRAY_JOB_ID}_${SLURM_ARRAY_TASK_ID}*" $LOG_DIR
+mv ATACpreAnalysisS1-${SLURM_ARRAY_JOB_ID}_${SLURM_ARRAY_TASK_ID}* $LOG_DIR
```

Comment:
Cool, this should work provided there are no change directory commands in the config that is being sourced. Last I checked this was indeed not the case